internal/feed/rss.go

```diff
@@ -158,8 +159,8 @@ func (j *RSSJob) processItem(item *gofeed.Item) *domain.Release {
 			if parentURL, _ := url.Parse(j.URL); parentURL != nil {
 				parentURL.Path, parentURL.RawPath = "", ""
 
-				// unescape the query params for max compatibility
-				escapedUrl, _ := url.QueryUnescape(parentURL.JoinPath(rls.DownloadURL).String())
+				downloadURL := sanitize.SanitizeURLEncoding(rls.DownloadURL)
```

Comment:
```suggestion
				downloadURL := sanitize.URLEncoding(rls.DownloadURL)
```
I would avoid adding the package name to the function